feature-libs/product-configurator/rulebased/styles/_configurator-overview-form.scss

```diff
@@ -228,105 +192,50 @@
             padding-block-end: 16px;
           }
 
-          border: solid 1px var(--cx-color-light);
-          background-color: var(--cx-color-background);
           font-size: 1rem;
           font-weight: normal;
           text-transform: uppercase;
-          position: sticky;
+          border: solid 1px var(--cx-color-light);
+          background-color: var(--cx-color-background);
+
+          @include forVersion(5.1) {
```

Comment:
This is already nested into a forVersion(5.1) include, so we don't need it, right?

Reply:
You are right! I will remove it.

Reply:
Done!